README.md

```diff
@@ -324,10 +324,50 @@ var path = await PdftronFlutter.saveDocument();
 Return a map object with values for position (bottom-left: `x1`, `y1`; top-right: `x2`, `y2`) and size (`width`, `height`) of the crop box for specified page. Values all have type `double`.
 
 ```dart
-var cropBox = await controller.getPageCropBox(1);
+var cropBox = await PdftronFlutter.getPageCropBox(1);
 print('The width of crop box for page 1 is: ' + cropBox.width.toString());
 ```
 
+### PdftronFlutter.setToolMode(String)
+
+To set the current tool mode (`Tools` constants in `lib/option.dart`).
+
+```dart
+ PdftronFlutter.setToolMode(Tools.annotationCreateEllipse);
+```
+
+### PdftronFlutter.setFlagForFields(List<`String`>, flag: int, flagValue: bool)
+
+Set a field flag value on one or more form fields.
+
+Params
+Name | Type | Description
+--- | ---| ---
+fieldNames | List<`String`> | A list of field names to be set
+flag | int | The flag to be set, one of the constants from `FieldFlags` in `lib/options.dart`
+flagValue | bool | To turn on/off the flag for the fields
+
+```dart
+ PdftronFlutter.setFlagForFields(['First Name', 'Last Name'], FieldFlags.Required, true);
+```
+
+### PdftronFlutter.setValueForFields(List<`Field`>)
```

Comment:
Should be `setValuesForFields` (values is plural).

Reply:
Maybe my English is bad, but i think it is single since we are only setting one value for each field. But anyway, if what you said correct, this naming change will cause similar functions (setValueForFields, setFlagForFields, setFlagForAnnotations, etc.) in RN and FL to change, or there would be an inconsistency. Should I update all of them?

Reply:
I had to think about it too. But it should be plural. Imagine you were giving one nut each to 7 squirrels. You'd say "Give the nuts to the squirrels", where nuts is plural because there are multiple nuts and multiple squirrels.

I think the only way value should be singular would if the same value was being applied to all fields. But this is not the case, there are many different values being assigned.

Reply:
prefer `setValueForFields` to have consistent naming with React Native, but fine with either

Reply:
I guess ideally we should change RN then. We could only document the new one, leave the old one in there to so we don't break customer code.

Reply:
sure